about-the-apprentices/2022/Terri.md

```diff
@@ -0,0 +1,13 @@
+# Hello there
+
+Hello everyone! My name is Terri and I'm a new apprentice at Sparkbox! Right now, I'm working remotely from Vermont, which means I'm on EST time and my preferred pronouns are she/hers.
+
+One of my favorite things about programming is it's creative and like solving puzzles. One thing I'm excited about learning is rebasing.
+
+## Icebreakers
+
+* What technology innovation made the most impact on your life
+The Personal Computer
+
+* Desktop, laptop, or tablet? Which do you prefer to use?
+Desktop for sure!!
```

Comment:
Add another exclamation point, two is simply not exiting enough!

Reply:
I just made this change, please review and approve if ok :)